index.html

```diff
@@ -44,152 +44,19 @@
-            <div id="extendedToolbarButtons"></div>
-
-            <a class="button icon-feedback" id="feedbackButton"></a>
-
-            <div id="sideToolbarContainer"></div>
-        </div>
-        <div id="videospace">
-            <div id="largeVideoContainer" class="videocontainer">
-                <div id="sharedVideo"><div id="sharedVideoIFrame"></div></div>
-                <div id="etherpad"></div>
-                <a target="_new"><div class="watermark leftwatermark"></div></a>
-                <a target="_new"><div class="watermark rightwatermark"></div></a>
-                <a class="poweredby" href="http://jitsi.org" target="_new">
-                    <span data-i18n="poweredby"></span> jitsi.org
-                </a>
-                <div id="dominantSpeaker">
-                    <div class="dynamic-shadow"></div>
-                    <img id="dominantSpeakerAvatar" src=""/>
-                </div>
-                <span id="remoteConnectionMessage"></span>
-                <div id="largeVideoWrapper">
-                    <video id="largeVideo" muted="true" autoplay></video>
-                </div>
-                <span id="localConnectionMessage"></span>
-                <span id="videoResolutionLabel" class="video-state-indicator moveToCorner">HD</span>
-                <span id="recordingLabel" class="video-state-indicator centeredVideoLabel">
-                    <span id="recordingLabelText"></span>
-                    <img id="recordingSpinner" class="recordingSpinner" src="images/spin.svg"></img>
-                </span>
-            </div>
-            <div class="filmstrip">
-                <div class="filmstrip__videos" id="remoteVideos">
-                    <span id="localVideoContainer" class="videocontainer">
-                        <div class="videocontainer__background"></div>
-                        <span id="localVideoWrapper">
-                            <!--<video id="localVideo" autoplay muted></video> - is now per stream generated -->
-                        </span>
-                        <audio id="localAudio" autoplay muted></audio>
-                        <div class="videocontainer__toolbar"></div>
-                        <div class="videocontainer__toptoolbar"></div>
-                        <div class="videocontainer__hoverOverlay"></div>
-                    </span>
-                    <audio id="userJoined" src="sounds/joined.wav" preload="auto"></audio>
-                    <audio id="userLeft" src="sounds/left.wav" preload="auto"></audio>
-                </div>
-            </div>
-
-        </div>
-    </div>
+    <div id="root"></div>
     <div id="keyboard-shortcuts" class="keyboard-shortcuts" style="display:none;">
         <div class="content">
             <ul id="keyboard-shortcuts-list" class="shortcuts-list">
             </ul>
         </div>
     </div>
+    <script src="libs/main.min.js"></script>
+    <script src="libs/app.bundle.min.js?v=139"></script>
```

Comment:
I'll think about the split between main.min.js and app.bundle.min.js tomorrow because I'd like us to try to avoid it.